internal/conf/logger/logger.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2023 EMQ Technologies Co., Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logger
+
+import (
+	"os"
+	"strings"
+
+	filename "github.com/keepeye/logrus-filename"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	Log     *logrus.Logger
+	LogFile *os.File
+)
+
+func init() {
```

Comment:
Why adding init as InitLogger already be invoked explicitly.

Reply:
In some unit test, `InitLogger` won't be called by `conf.init` as expected.

Reply:
It looks like current implementation will init logger twice. Can we just remove InitLogger in conf?

Reply:
updated.